lib/streamlit/config.py

```diff
@@ -296,6 +293,56 @@ def _global_unit_test():
     type_=int,
 )
 
+
+# Config Section: Logger #
+_create_section("logger", "Settings to customize Streamlit log messages.")
+
+
+@_create_option("logger.datetimeFormat", type_=str)
+def _logger_datetime_format():
```

Comment:
This feels over-customizable. I why would anyone want to change the timestamp format? I'd rather provide a way to turn timestamps on/off instead:

```
[logger]
showTimestamps = false  # Default == true
```

Reply:
This was because I was just passing through the options the python logger currently provides. It could be unlikely users would care to change the format.